### PR TITLE
Changes for PowerPC in partitioning_raid.pm

### PR DESCRIPTION
**Pull request by michelmno**
because PowerPC need a PReP partition
and not a RAID /boot partition.

Use "0x41 PReP Boot" not "0x00 GPT PReP Boot"

Signed-off-by: Michel Normand <normand@linux.vnet.ibm.com>

prereq needles from https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/246

### CODE REVIEW

**Comment by okurz** [comment]
Hm, I'm not so sure about this. We don't see the according problem in our SLE tests. Could you link jobs that fail that should be fixed by this? Is this a recently observed problem?

**Comment by michelmno** [comment]
@okurz , 
I just submitted a clone on o3 at https://openqa.opensuse.org/tests/455557
to show that  ppc64le openSUSE is failing for RAID0 test with default openQA scripts and needles.

My proposed changes have been validated on an internal openQA instance (in IBM) you are not able to access.

File: tests/installation/partitioning_raid.pm [hunk 1]
```diff
@@ -163,10 +163,9 @@ sub run {
             assert_screen 'partitioning-type';
             send_key 'alt-n';
         }
-        assert_screen 'partitioning-size';
+        assert_screen 'partition-size';
```
Comment (okurz) [inline tests/installation/partitioning_raid.pm]: Our current SLE ppc64le tests have needles with the tag `partitioning-size` so I would prefer to keep this and the following code as is.

**Comment by michelmno** [comment]
As suggested by @okurz  I am not changing anything in the script and so closing this PR.

The RAID* tests passed for openSUSE TW after addition of missing needles directly via WEBUI.
https://openqa.opensuse.org/tests/overview?build=20170720&version=Tumbleweed&groupid=4&distri=opensuse

**Comment by okurz** [comment]
great! thank you

**Pull request closed without merging**